Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:buster
+
+RUN apt-get update \
+ && apt-get install --yes \
+      wget \
+      python3-distutils \
+      python3-scapy \
+      pciutils \
+ && rm -rf /var/apt/lists/*
+
+RUN cd tmp \
+ && wget https://trex-tgn.cisco.com/trex/release/v2.65.tar.gz \
+ && tar -xf v2.65.tar.gz \
+ && rm v2.65.tar.gz \
+ && mv v2.65 trex
+
+COPY traffic-configurations/veth_cfg.yaml /etc/trex_cfg.yaml
+COPY traffic-configurations/*.yaml /tmp/trex/cfg/
+COPY traffic-patterns/dns_customer_ip.yaml /tmp/trex/cap2/dns_customer_ip.yaml
+
+CMD exec /bin/bash
```

Comment:
The `exec` here is useless. The `CMD` can just be `bash` or `/bin/bash` if we want to be especially verbose. However, the default command is already bash, so you can just remove the `CMD` entirely.

Reply:
Removed CMD & everything worked fine.